### DISCUSSION TITLE
炎 is a Chinese word not a Japanese word

### DISCUSSION THREAD

**Discussion by lakkiy** [honojs/hono#3525] [General]
Hi, I just read the blog of hono:

https://blog.cloudflare.com/the-story-of-web-framework-hono-from-the-creator-of-hono/

And want to mention that 炎 is a Chinese word not a Japanese word

The character "炎" is originally a Chinese character, not exclusively a Japanese one. It originated in China and is part of the common Chinese character set. However, it has also been adopted into the Japanese writing system as one of the kanji characters, with its own pronunciation and usage in the Japanese language. Therefore, while "炎" is used in Japanese and pronounced as "ほのお" (Hono), it is still fundamentally a Chinese character that has been borrowed and adapted into Japanese.

**Comment by toshi-kuji** [comment]
I understand your point but nobody is saying the character is Japanese. The word 炎 read as Hono is Japanese.

**Reply by lakkiy** [reply]
Thanks, got it!
